go.mod

```diff
@@ -48,7 +49,7 @@ require (
 	github.com/prometheus/common v0.2.0 // indirect
 	github.com/prometheus/procfs v0.0.0-20190209105433-f8d8b3f739bd // indirect
 	github.com/rogpeppe/go-internal v1.2.2 // indirect
-	github.com/spf13/afero v1.2.1 // indirect
+	github.com/spf13/afero v1.2.1
```

Comment:
right now we are using `afero` just for our table tests in the `package_test.go`, right?

Reply:
yes